chain/client/src/test_utils.rs

```diff
@@ -557,10 +566,13 @@ pub fn setup_mock_all_validators(
     check_block_stats: bool,
     peer_manager_mock: Box<
         dyn FnMut(
+            // Peer validators
             &[(Addr<ClientActor>, Addr<ViewClientActor>)],
+            // Validator that sends the message
             AccountId,
+            // The message itself
             &PeerManagerMessageRequest,
-        ) -> (PeerManagerMessageResponse, bool),
+        ) -> (PeerManagerMessageResponse, /*perform default*/ bool),
```

Comment:
```suggestion
        ) -> (PeerManagerMessageResponse, /* perform default */ bool),
```

At least that the common formatting used in C/C++